fusillade-api.yml

```diff
@@ -1380,6 +1380,69 @@ paths:
           description: Unexpected error.
       tags:
         - resource
+  /v1/resource/{resource_type_name}/id/{resource_id}/members:
+    get:
+      summary: List all members that have access to this resource.
+      description: List all members that have access to this resource. Members include users, and groups with any
+        level of access to this resource.
+      operationId: fusillade.api.resource.resource_type_name.id.resource_id.members.get
+      security:
+        - BearerAuth: []
+      parameters:
+        - $ref: '#/components/parameters/resource_type_name'
+        - $ref: '#/components/parameters/resource_id'
+        - $ref: '#/components/parameters/next_token'
+        - $ref: '#/components/parameters/per_page'
+      responses:
+        200:
+          description: All results have been returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/members'
+        206:
+          description: All results have been returned.
```

Comment:
should this be "more results are available"

Reply:
will fix